### PR TITLE
Edited component menu in UI builder

### PR DESCRIPTION
**Pull request by vyhnalekl**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ yarn change`

#### Description of changes

Changed the component list in left side menu into a tree with logical categories. The categories are defined in `config.tsx`. Also, he preview pane was removed.
The bottom left tree (displaying tree of components that are in iframe) is styled differently, there is a `Devider `added between these two trees. Also, the bottom tree has always expanded all of it's items.

### CODE REVIEW

**Comment by dzearing** [comment]
PRs seem to be stuck. Being investigated.

File: packages/fluentui/react-builder/src/components/ComponentTree.tsx [hunk 1]
```diff
@@ -42,24 +52,40 @@ const jsonTreeToTreeItems: (
         ...(selectedComponentId === tree.uuid && {
           background: '#ffc65c',
           color: '#444',
+          borderBottomLeftRadius: '1.2vh',
```
Comment (mnajdova) [inline packages/fluentui/react-builder/src/components/ComponentTree.tsx:55]: please use `rem`

File: packages/fluentui/react-builder/src/components/ComponentTree.tsx [hunk 2]
```diff
@@ -106,10 +132,55 @@ export const ComponentTree: React.FunctionComponent<ComponentTreeProps> = ({
...
+  const titleRenderer = (Component, { content, expanded, hasSubtree, ...rest }) => {
+    return (
+      <>
+        {rest['level'] !== 1 ? (
+          <span
+            style={{
+              paddingRight: '1rem',
+              borderLeft: '1px solid #eee',
+              borderBottom: '1px solid #eee',
+              borderBottomLeftRadius: '1.2vh',
```
Comment (mnajdova) [inline packages/fluentui/react-builder/src/components/ComponentTree.tsx:152]: `rem` or `px` please

File: packages/fluentui/react-builder/src/components/List.tsx [hunk 3]
```diff
@@ -30,42 +27,75 @@ export const List: React.FunctionComponent<ListProps> = ({ onDragStart, style })
...
+            items: supportedComponents
+              .filter(info => COMPONENT_GROUP[key].includes(info.isChild ? info.parentDisplayName : info.displayName))
+              .map(info => ({
+                id: info.displayName,
+                title: (
+                  <Box
+                    key={info.displayName}
+                    onMouseDown={handleMouseDown(info)}
+                    styles={{
+                      padding: '0.25em 0.75em',
```
Comment (mnajdova) [inline packages/fluentui/react-builder/src/components/List.tsx:67]: `px` or `rem` please :)

**Pull request merged**